### PR TITLE
fix: Improve mixins

### PR DESCRIPTION
**Pull request by Dariia**

### CODE REVIEW

File: src/modules/ui/shared.scss [hunk 1]
```diff
@@ -1,5 +1,12 @@
 @import 'media-query-mixins';
 
+$breakpoints: (
```
Comment (rudnitskih) [inline src/modules/ui/shared.scss]: @Dariia Unused?

**Comment by rudnitskih** [comment]
@Dariia let's rename `media-query-mixins.scss`. I don't have a good name, maybe `conditions.scss` ?

File: src/modules/ui/shared.scss (hunk 1 above)
Comment (Dariia) [reply src/modules/ui/shared.scss]: right, will remove

**Comment by Dariia** [comment]
> @Dariia let's rename `media-query-mixins.scss`. I don't have a good name, maybe `conditions.scss` ?

File: src/modules/ui/conditions.scss [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+}
+
+@mixin apply($mixins...) {
+  $query: '';
+  $direction: false;
+
+  @each $mixin in $mixins {
+    $query: $query + $mixin;
+
+    @if ($mixin == direction-query('ltr')) {
```
Comment (rudnitskih) [inline src/modules/ui/conditions.scss]: @BrooklynKing @bodia-uz we don't like this part

File: src/modules/ui/conditions.scss [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+ *   @include apply(max-width-550(), ltr())
+ *   @include apply(max-width-550(), rtl())
+ */
+
+@function max-width($value) {
+  @return '[data-playable-max-width~="#{$value}"]'
+}
+
+@function direction-query($direction) {
+  @return if($direction, '[data-playable-dir="#{$direction}"]', '')
```
Comment (bodia-uz) [inline src/modules/ui/conditions.scss]: not sure if this condition is required here. If we use `direction-query`, its strange to not pass `$direction`

File: src/modules/ui/conditions.scss (hunk 2 above)
Comment (bodia-uz) [reply src/modules/ui/conditions.scss]: is this for next code?
```scss
@if ($direction) {
  direction: $direction;
}
```

Mb we should not set `direction` rule in this `query`.
And if we need it in some block, set it manually under query.
```scss
@include apply(ltr()) {
  margin-right: 15px;
}	
@include apply(rtl()) {
  direction: 'rtl'
  margin-left: 15px;
}
```

Comment (Dariia) [reply src/modules/ui/conditions.scss]: We have quite many places to put direction and it's can be not obvious or forgotten:
image

File: src/modules/ui/conditions.scss (hunk 3 above)
Comment (Dariia) [reply src/modules/ui/conditions.scss]: agree, removed

File: src/modules/ui/conditions.scss (hunk 2 above)
Comment (bodia-uz) [reply src/modules/ui/conditions.scss]: Not all of these lines requires to add `direction: rtl`.
You need only one per `live-indicator`.
Other ones are only about `margin-left` and `margin-right`

**Pull request merged**